pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -96,11 +96,14 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 		return nil, fmt.Errorf("unable to parse PodEvictionTimeout: %v", err)
 	}
 
+	// Defaults are tested in TestAPIServerDefaults
 	server := apiserveroptions.NewAPIServer()
+	// Adjust defaults
 	server.EventTTL = 2 * time.Hour
 	server.ServiceClusterIPRange = net.IPNet(flagtypes.DefaultIPNet(options.KubernetesMasterConfig.ServicesSubnet))
 	server.ServiceNodePortRange = *portRange
 	server.AdmissionControl = strings.Join(AdmissionPlugins, ",")
+	server.EnableLogsSupport = false // don't expose server logs
```

Comment:
this is the only functional change in config. other things are set to make previously implicit things explicit